pyproject.toml

```diff
@@ -27,7 +27,10 @@ pyshacl = "^0.19.0"
 alembic = "^1.8.0"
 Flask = "^2.1.2"
 Flask-API = "^3.0.post1"
+BAC0 = "^22.9.21"
+netifaces = "^0.11.0"
 rfc3987 = "^1.3.8"
+openpyxl = "^3.0.10"
```

Comment:
I used `openpyxl` because that is what is used in Pandas and it seemed to work well and had minimal other dependencies. I'll take a look at some of the others just in case they might offer broader/more robust support